src/test/java/org/jsmart/zerocode/samplesjunit5/jupiter/JUnit5Test.java

```diff
@@ -0,0 +1,27 @@
+package org.jsmart.zerocode.samplesjunit5.jupiter;
+
+import org.jsmart.zerocode.samplesjunit5.jupiter.extension.ExtensionA;
+import org.jsmart.zerocode.samplesjunit5.jupiter.extension.ExtensionB;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static java.lang.System.out;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith({ExtensionA.class, ExtensionB.class})
```

Comment:
can you put a comment on the class file why `ExtensionA and B` are there?

Reply:
That's just to show the users that the Junit5 tests run with extension enabled and that can be observed in the log too.